Makefile

```diff
@@ -281,12 +279,8 @@ test-examples-kustomize-against-HEAD: $(MYGOBIN)/kustomize $(MYGOBIN)/mdrip
 	./hack/testExamplesAgainstKustomize.sh HEAD
 
 .PHONY:
-test-examples-kustomize-against-3.9: $(MYGOBIN)/mdrip
-	./hack/testExamplesAgainstKustomize.sh v3.9.3
-
-.PHONY:
-test-examples-kustomize-against-3.8: $(MYGOBIN)/mdrip
```

Comment:
Please convert this into `-3.10`, a test against `v3.10.0`

Reply:
done